sdk/core/src/request_options/accept.rs

```diff
@@ -0,0 +1,29 @@
+use crate::headers::{self, Header};
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub struct Accept<'a>(&'a str);
```

Comment:
Can we add some documentation to these header types?

Reply:
Yes, done. For the headers also available in the HTTP crate I just copied some docs from there. For other I went for the text in the Kusto docs.